{
  "uid": "dbbaed192f51daa0",
  "name": "test_broken",
  "fullName": "pytest_Allure#test_broken",
  "historyId": "36124afcea75fcacc06774865a38248a",
  "time": {
    "start": 1627290490909,
    "stop": 1627290490909,
    "duration": 0
  },
  "status": "broken",
  "statusMessage": "Exception: oops",
  "statusTrace": "def test_broken():\n>       raise Exception(\"oops\")\nE       Exception: oops\n\npytest_Allure.py:9: Exception",
  "flaky": false,
  "newFailed": false,
  "beforeStages": [],
  "afterStages": [],
  "labels": [
    {
      "name": "suite",
      "value": "pytest_Allure"
    },
    {
      "name": "host",
      "value": "LAPTOP-V2RHK3K3"
    },
    {
      "name": "thread",
      "value": "20060-MainThread"
    },
    {
      "name": "framework",
      "value": "pytest"
    },
    {
      "name": "language",
      "value": "cpython3"
    },
    {
      "name": "package",
      "value": "pytest_Allure"
    },
    {
      "name": "resultFormat",
      "value": "allure2"
    }
  ],
  "parameters": [],
  "links": [],
  "hidden": false,
  "retry": false,
  "extra": {
    "severity": "normal",
    "retries": [],
    "categories": [
      {
        "name": "Test defects",
        "matchedStatuses": [],
        "flaky": false
      }
    ],
    "tags": []
  },
  "source": "dbbaed192f51daa0.json",
  "parameterValues": []
}